lib/resources/postgres_session.rb

```diff
@@ -31,6 +31,10 @@ class PostgresSession < Inspec.resource(1)
       describe sql.query('SELECT * FROM pg_shadow WHERE passwd IS NULL;') do
         its('output') { should eq('') }
       end
+
+      describe sql.psql('SHOW port') do
```

Comment:
I do not agree with creating a `psql` method. No matter how the query is performed, even `SHOW port` is still a query so the `query` method will suffice.